numpy/_core/src/multiarray/item_selection.c

```diff
@@ -2045,6 +2062,7 @@ PyArray_SearchSorted(PyArrayObject *op1, PyObject *op2,
         Py_DECREF(dtype);
         return NULL;
     }
+    
```

Comment:
```suggestion
```
Please avoid this type of change (and this case, two lines are too much in NumPy style)